SPECS/compiler-rt/compiler-rt.spec

```diff
@@ -0,0 +1,72 @@
+%global maj_ver 12
+%global min_ver 0
+%global patch_ver 1
+
+%global compiler_rt_srcdir %{name}-%{version}.src
+
+Summary:        LLVM compiler support routines
+Name:           compiler-rt
+Version:        %{maj_ver}.%{min_ver}.%{patch_ver}
+Release:        1%{?dist}
+License:        Apache 2.0 WITH exceptions
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://compiler-rt.llvm.org
+Source0:        https://github.com/llvm/llvm-project/releases/download/llvmorg-%{version}/%{compiler_rt_srcdir}.tar.xz
+BuildRequires:  cmake
+BuildRequires:  llvm-devel = %{version}
+Requires:       llvm
+
+%description
+The compiler-rt project consists of several related runtime libraries for interfacing
+with the clang compiler infrastructure. Builtins is a library for providing low-level
+target-specific hooks required by code generation and other runtime components. Sanitizer
+consists of several sanitizers such as the AddressSanitizer that can include instrumentation
+into compiled binaries. Profile is a library used to collect coverage information and
+BlocksRuntime is an implementation of Apple "blocks" interface.
+
+%prep
+%setup -q -n %{compiler_rt_srcdir}
+
+%build
+# Disable symbol generation
+export CFLAGS="`echo " %{build_cflags} " | sed 's/ -g//'`"
+export CXXFLAGS="`echo " %{build_cxxflags} " | sed 's/ -g//'`"
+
+mkdir -p build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=%{_prefix}   \
+      -DCMAKE_BUILD_TYPE=Release          \
+      -Wno-dev ..
+
+%make_build
+
+%install
+cd build
+%make_install
+
+mkdir -p %{buildroot}%{_libdir}/clang/%{version}/share
+mv -v %{buildroot}%{_datadir}/*list.txt  %{buildroot}%{_libdir}/clang/%{version}/share/
+
+mkdir -p %{buildroot}%{_libdir}/clang/%{version}/lib/linux
+mv -v %{buildroot}%{_prefix}/lib/linux/*clang_rt* %{buildroot}%{_libdir}/clang/%{version}/lib/linux
+
+%files
+%defattr(-,root,root)
+%license LICENSE.TXT
+
+%{_includedir}/fuzzer
+%{_includedir}/profile
+%{_includedir}/sanitizer
+%{_includedir}/xray
+%{_libdir}/clang/%{version}/lib/linux/*
+%{_libdir}/clang/%{version}/share/*
+%{_bindir}/hwasan_symbolize
+
+%changelog
+* Wed Dec 07 2022 Adam Schwab <adschwab@microsoft.com>
+- Influenced by Fedora's spec from version fc35 (license: MIT)
```

Comment:
Since these two change log entries are for the same version, they should be collapsed into a single entry.

Reply:
I wanted to do this but wasn't sure it would pass the checker, I guess it has to be exactly:
```
- Original version for CBL-Mariner. Influenced by Fedora's spec from version fc35 (license: MIT)
```
This should pass the test, I can change it to this.